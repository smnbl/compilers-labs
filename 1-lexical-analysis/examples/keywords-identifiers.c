return
if else while for

foo foo123
foo_bar
1foo
