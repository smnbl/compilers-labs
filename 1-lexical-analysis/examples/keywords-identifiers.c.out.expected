1:1 -> 1:7          return              RETURN              
2:1 -> 2:3          if                  IF                  
2:4 -> 2:8          else                ELSE                
2:9 -> 2:14         while               WHILE               
2:15 -> 2:18        for                 FOR                 
4:1 -> 4:4          foo                 IDENTIFIER          
4:5 -> 4:11         foo123              IDENTIFIER          
5:1 -> 5:8          foo_bar             IDENTIFIER          
6:1 -> 6:2          1                   INT_LITERAL         
6:2 -> 6:5          foo                 IDENTIFIER          
